esmvaltool/utils/color-tables/show_color_tables.py

```diff
@@ -0,0 +1,40 @@
+"""
+Utility script for inspecting ncl color tables
+
+"""
+import glob
+import os
+
+from jinja2 import Template
+
+PATH_TO_COLORTABLES = "../../../esmvaltool/diag_scripts/shared/plot/rgb"
```

Comment:
Maybe you can use something like
```
from esmvaltool.diag_scripts.shared.plot import __file__ as plot_path
PATH_TO_COLORTABLES = os.path.join(os.path.dirname(plot_path), 'rgb')
```
to make the script work from other locations.